var/spack/repos/builtin/packages/sirius/package.py

```diff
@@ -299,6 +299,9 @@ def cmake_args(self):
                 ]
             )
 
+        if "^cray-libsci" in spec:
+            args.extend(["-DUSE_CRAY_LIBSCI=On"])
```

Comment:
```suggestion
            args.append(self.define("USE_CRAY_LIBSCI", "ON"))
```

untested, but for consistency with the rest?

Reply:
Thanks, I've changed it accordingly.